assets/scss/base/djaodjin-dashboard.scss

```diff
@@ -113,7 +114,7 @@ $dashboard-sidebar-color:    white;
 }
 
 .dashboard-nav-menu {
-  height: calc(100vh - (#{$dashboard-nav-footer-height} + #{$navbar-height}));
+  height: calc(100vh - (#{$dashboard-nav-footer-height} + #{$dashboard-sidebar-logo-height}));
```

Comment:
This is strange. if `navbar-height` is not equal to `dashboard-sidebar-logo-height` then the sidebar logo and menu items in the navbar will not be properly aligned, correct?

Reply:
No, we have two full height vertical sides. Changing logo to the height equal to navbar height would reduce the logo size.

Reply:
Yes. Reducing the logo size is what we do [here]
(https://github.com/djaodjin/djaoapp/commit/be843468e65d034650b3005072a611cde0859ad4#diff-b3296c59187bdddcc67abbf04e53e9e3R54). I will merge anyway. We can figure something better later.